### PR TITLE
[th2-1515] removed dependency on Bintray

### PR DESCRIPTION
**Pull request by Maximus905**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,11 +1,10 @@
 plugins {
+    id 'java'
+    id 'java-library'
+	id 'application'
```
Comment (OptimumOpium) [inline build.gradle:4]: Please, format plugins

File: build.gradle [hunk 2]
```diff
@@ -24,9 +23,14 @@ repositories {
...
-        url 'https://dl.bintray.com/exactpro/th2'
-    }
+		name 'Sonatype_snapshots'
+		url 'https://s01.oss.sonatype.org/content/repositories/snapshots/'
+	}
+
+	maven {
+		name 'Sonatype_releases'
+		url 'https://s01.oss.sonatype.org/content/repositories/releases/'
+	}
```
Comment (OptimumOpium) [inline build.gradle:33]: Please, format repository declaration

**Comment by OptimumOpium** [comment]
Also, please remove bintray keys from the Docker file

File: build.gradle (hunk 1 above)
Comment (Maximus905) [reply build.gradle]: done

File: build.gradle (hunk 2 above)
Comment (Maximus905) [reply build.gradle]: done

**Comment by Maximus905** [comment]
bintray keys removed

**Review by OptimumOpium — APPROVED** [review]

**Pull request merged**